src/_internals/item/index.tsx

```diff
@@ -30,6 +30,26 @@ const StyledItem = styled(Item)`
     background: none;
   }
 
+  &.kirk-item--wrappable {
+    flex-flow: wrap;
+    padding-bottom: ${space.m};
+  }
+
+  &.kirk-item--wrappable .kirk-item-leftWrapper {
```

Comment:
nit: you could use **Sass** syntax by nesting the classes attached to the same component. 
It would increase readability when we go for a refactoring to **css-in-js** syntax

Reply:
Done